contracts/upgradeable_contracts/ChaiConnector.sol

```diff
@@ -185,6 +185,9 @@ contract ChaiConnector is Ownable, ERC20Bridge, TokenSwapper {
 
         receiver.call(abi.encodeWithSelector(ON_TOKEN_TRANSFER, address(this), interest, ""));
 
+        // Additional constant is not needed here, since we allow to withdraw only extra part of chai balance,
```

Comment:
```suggestion
        // Additional constant to tolerate the DAI balance deposited to the Chai token is not needed here, since we allow to withdraw only extra part of chai balance,
```